data/json/items/tool/misc.json

```
@@ -937,5 +937,18 @@
       { "type": "firestarter", "moves": 1000, "moves_slow": 25000, "need_sunlight": true }
     ],
     "flags": [ "FIRESTARTER", "THERMOMETER" ]
+  },
+  {
+    "id": "pocketwatch_crude",
+    "type": "GENERIC",
+    "category": "tools",
+    "copy-from": "pocketwatch",
+    "name": { "str": "steel pocket watch", "str_pl": "steel pocket watches" },
+    "description": "An old-fashioned pocket watch, useful for telling the time and not much else.",
+    "looks_like": "pocketwatch",
+    "price": "250 USD",
+    "price_postapoc": "1 USD",
+    "to_hit": -3,
```

```suggestion
    "copy-from": "pocketwatch",
    "name": { "str": "steel pocket watch", "str_pl": "steel pocket watches" },
    "description": "An old-fashioned pocket watch, useful for telling the time and not much else.",
    "looks_like": "pocketwatch",
    "price": "250 USD",
    "price_postapoc": "1 USD",
```
Those two bits are same as watch it inherits from so don't need them, will cause it to complain about updating assigned values on load otherwise.